routes/v1.js

```diff
@@ -33,12 +34,32 @@ apiRoutes.route('/verify/:verificationId')
   .get(UserController.verifyAccount);
 
 apiRoutes.route('/articles')
-  .post(Auth.verifyUser, isUserVerified,
+  .post(
+    Auth.verifyUser, isUserVerified,
```

Comment:
@Dubby20 let these be on different lines

Reply:
Alright